### DISCUSSION TITLE
posts/2021-my/%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#278] [Announcements]
# posts/2021-my/%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4

人若无名，专心练剑

http://www.iocaop.com/posts/2021-my/%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4.html

**Comment by lzcxfz** [comment]
测试
